src/clims/models/transition.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import
+
+from django.db import models
+from sentry.db.models import (Model, FlexibleForeignKey, sane_repr)
+
+
+class TransitionType(object):
+    MOVE = 0
+    SPAWN = 1
+
+
+class Transition(Model):
+    """
+    Represents a Transition for a Substance.
+    """
+    __core__ = True
+
+    BACKEND_CAMUNDA = 'camunda'
+
+    # Added because it seems to be used in all CLIMS models.
+    # TODO: check if it is actually needed?
+    organization = FlexibleForeignKey('sentry.Organization')
```

Comment:
I think that this doesn't need to be in the model, although a transition is under an organization in the end. The reason is that the transitions always happen in the scope of a `WorkBatch` and they relate to the organization.

Reply:
Removing